.github/workflows/build-and-push-component.yaml

```diff
@@ -24,34 +20,24 @@ on:
         required: true
         type: string
         description: 'The build timestamp to be used for binaries.'
+      bake_target:
+        required: true
+        type: string
+        description: 'Docker Bake target name.'
+      cache-name-prefix:
+        required: true
+        type: string
+        description: 'A prefix for image names and digests to be cached temporarily.'
+      post_image_tags:
+        required: false
+        type: boolean
+        description: |
+          If set to true the image tags pushed to the repository are posted as comment for the Pull Request.
+          Only works if the event type is `pull_request`.
 
 jobs:
-  build-vars:
-    runs-on: ubuntu-latest
-    outputs:
-      digests-cache-name: ${{ steps.build-vars.outputs.digests-cache-name }}
-    steps:
-      - name: Set build variables
-        id: build-vars
-        run: |
-          
-          ##
-          ## Extract the image name
-          ##
-          ##    $ basename ghcr.io/openclarity/vmclarity-ui-dev
-          ##    vmclarity-ui-dev
-          ##
-          image_name="$(basename ${{ inputs.image_name }})"
-          
-          ##
-          ## Set digests cache name
-          ##
-
-          echo "digests-cache-name=digests-${{ github.run_id }}-${image_name}" >> "$GITHUB_OUTPUT"
```

Comment:
can this be preserved to avoid having to specify it manually?

Reply:
CI changes will be part of another PR